fstest/test_all/config.yaml

```diff
@@ -499,3 +499,6 @@ backends:
  - backend:  "ulozto"
    remote:   "TestUlozto:"
    fastlist: false
+ - backend:  "icloud"
```

Comment:
The backend is called `iclouddrive` isn't it?

Reply:
the backend yes but it lives under icloud as im planning photos etc also under it

Reply:
This needs to be the exact name of the backend for the tests to work.